README.md

```diff
@@ -1,45 +1,62 @@
-# Coder Monorepo
+# Welcome to CODENAUTE 🧑🚀
 
 We use [pnpm](https://pnpm.io/workspaces) built-in support to manage our monorepository.
 
-## Install dependencies
+## IDE configuration
 
-To install all dependencies, run:
+It is highly recommended to be working with VSCode, an IDE that does not need to be presented. Internally, we use a set of code extensions enabling a minimum of code standardization, making the life of many developers more enjoyable. Those extensions are given in `.vscode/extensions.json`, and can be downloaded directly via the VSCode extension store. This goes hand and hand with properly configured VSCode workspace settings, available in `.vscode/settings.json`.
 
-`make install`
+## Requirements
 
-If you want to hard install all dependencies, run:
+To use this repository, you need to have the following tools installed:
 
-`make hard-install`
+- [Node.js](https://nodejs.org/en/) (v16.17.1)
+- [pnpm](https://pnpm.io/installation) (v7.16.0)
+- [Makefile](https://perso.univ-lyon1.fr/jean-claude.iehl/Public/educ/Makefile.html)
+- [Docker](https://docs.docker.com/get-docker/) (v20.10.8)
+- [Docker Compose](https://docs.docker.com/compose/install/) (v1.29.2)
 
-## Start applications
+## Install dependencies
 
-To start all applications, run:
+To install all dependencies, run:
 
-`make start`
+```
+make install
+```
 
-To start server application, run:
+If you want to **hard install** all dependencies, you can run:
 
-`make server`
+```
+make hard-install
+```
 
-To start client application, run:
+## Start applications
 
-`make client`
+After install all dependencies, we need start the application, you can do it running:
```

Comment:
Maybe that's better to explane that a dev will be more comfortable to run `make start-server` and `make start-client` on two terminal windows, because start all is not very useful (no debug logs here)

Reply:
Done